src/core/mavlink_parameters.h

```diff
@@ -546,6 +552,12 @@ class MAVLinkParameters {
     std::mutex _all_param_mutex{};
 
     // dl_time_t _last_request_time = {};
+
+    std::map<std::string, ParamValue> _param_server_store;
```

Comment:
A `std::vector<std::pair<std::string, ParamValue>>` might be faster for small numbers of params.

Reply:
Noted. I have left it as a map at current just because of it's easy lookup api compared to a pair.